### DISCUSSION TITLE
warning-signs-of-fake-lego-porsche-911-rsr/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-personal#10] [Comments]
# warning-signs-of-fake-lego-porsche-911-rsr/

I recently received a LEGO Porsche 911 RSR as a gift, and during the build process a few odd things happened. Pieces didn’t look quite right, the packaging wasn’t what I was used to… it’s a fake! However, it’s such a high quality fake that I didn’t realise until 1/3 of the way through, and I decided to keep it. Here’s a few things to check to avoid being scammed.

https://jakelee.co.uk/warning-signs-of-fake-lego-porsche-911-rsr/

**Comment by jkpe** [comment]
Hi Jake,
Thanks for sharing, this was an interesting read.

I found myself conflicted having become a bit obsessed with LEGO recently and now I really want the Titanic set. You can buy replicas for 1/6 the price. Initially it put things in perspective, why the hell would I paying £600 for some plastic that can clearly be manufactured for so little. But after a while I realised that's not what you're paying for, you're paying for the time and effort the designers have spent to create this magnificent work of art, and you're paying to support LEGO so they can continue to come up with amazing sets.

It remains on my wish list and hoping to pick it up from the store soon!

Take care,
Jack

**Reply by JakeSteam** [reply]
Completely agree with everything you said. It's hard intentionally choosing to pay significantly more for an almost identical product, but the responsible thing to do if you can in this case!
